src/main/java/com/cchess/game/ws/MessageService.java

```diff
@@ -0,0 +1,28 @@
+package com.cchess.game.ws;
+
+import com.cchess.game.user.UserDto;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.messaging.simp.SimpMessagingTemplate;
+import org.springframework.stereotype.Service;
+
+@Service
+@Slf4j
+@RequiredArgsConstructor
+public class MessageService {
+
+    private final SimpMessagingTemplate messagingTemplate;
+
+    public void notifyPlayerJoin(String roomId, UserDto user) {
+        sendMessage("/room/" + roomId, user);
+    }
+
+    private void sendMessage(String destination, Object payload) {
+        try {
+            messagingTemplate.convertAndSend(destination, payload);
+            log.info("Message sent to {}", destination);
```

Comment:
Check if INFO log level here is appropriate? Learn when to use which log level, this's really important